content/docs/conditional-rendering.md

```diff
@@ -1,243 +1,245 @@
 
   render() {
-    const isLoggedIn = this.state.isLoggedIn;
-    let button;
+    const estaConectado = this.state.estaConectado;
+    let boton;
 
-    if (isLoggedIn) {
-      button = <LogoutButton onClick={this.handleLogoutClick} />;
+    if (estaConectado) {
+      boton = <BotonCierreSesion onClick={this.manejarClickCierreSesion} />;
     } else {
-      button = <LoginButton onClick={this.handleLoginClick} />;
+      boton = <BotonInicioSesion onClick={this.manejarClickInicioSesion} />;
     }
 
     return (
       <div>
-        <Greeting isLoggedIn={isLoggedIn} />
-        {button}
+        <Saludo estaConectado={estaConectado} />
+        {boton}
       </div>
     );
   }
 }
 
 ReactDOM.render(
-  <LoginControl />,
-  document.getElementById('root')
+  <GestionInicioSesion />,
+  document.getElementById('raiz')
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Pruébalo en CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+Si bien declarar una variable y usar una sentencia `if` es una buena forma de renderizar condicionalmente un componente, a veces podrías querer usar una sintaxis más corta. Hay algunas formas de hacer condiciones $INLINE en JSX, explicadas a continuación.
 
-### Inline If with Logical && Operator
+### If $INLINE con operador lógico &&
 
-You may [embed any expressions in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) by wrapping them in curly braces. This includes the JavaScript logical `&&` operator. It can be handy for conditionally including an element:
+Puedes [embeber cualquier expresión en JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) envolviéndola en llaves. Esto incluye al operador lógico `&&` de JavaScript. Puede ser ùtil para incluir condicionalmente un elemento:
 
 ```js{6-10}
-function Mailbox(props) {
-  const unreadMessages = props.unreadMessages;
+function Buzon(props) {
+  const mensajesNoLeidos = props.mensajesNoLeidos;
   return (
     <div>
-      <h1>Hello!</h1>
-      {unreadMessages.length > 0 &&
+      <h1>¡Hola!</h1>
+      {mensajesNoLeidos.length > 0 &&
         <h2>
-          You have {unreadMessages.length} unread messages.
+          Tienes {mensajesNoLeidos.length} mensajes sin leer.
         </h2>
       }
     </div>
   );
 }
 
-const messages = ['React', 'Re: React', 'Re:Re: React'];
+const mensajes = ['React', 'Re: React', 'Re:Re: React'];
 ReactDOM.render(
-  <Mailbox unreadMessages={messages} />,
-  document.getElementById('root')
+  <Buzon mensajesNoLeidos={mensajes} />,
+  document.getElementById('raiz')
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ozJddz?editors=0010)
+[**Pruébalo en CodePen**](https://codepen.io/gaearon/pen/ozJddz?editors=0010)
```

Comment:
Please update the code in lines `127-147` to match this codepen link as much as possible. I think it's fine to change the content of the JSX but not the component names neither the root node.